show-changes

```diff
@@ -38,6 +39,16 @@ for section in security bugfixes changes updates; do
       exit 1
   esac
   echo
+  if [ "${section}" = security ]; then
+      FROM_KERNEL=$(git -C coreos-overlay show "${OLD}":sys-kernel/coreos-kernel/ | grep -m 1 'coreos-kernel-.*.ebuild' | cut -d - -f 3 | cut -d . -f 1-3)
+      TO_KERNEL=$(git -C coreos-overlay show "${NEW}":sys-kernel/coreos-kernel/ | grep -m 1 'coreos-kernel-.*.ebuild' | cut -d - -f 3 | cut -d . -f 1-3)
```

Comment:
```suggestion
      FROM_KERNEL=$(git -C coreos-overlay show "${OLD}":sys-kernel/coreos-kernel/ | grep -m 1 'coreos-kernel-.*\.ebuild' | cut -d - -f 3 | cut -d . -f 1-3)
      TO_KERNEL=$(git -C coreos-overlay show "${NEW}":sys-kernel/coreos-kernel/ | grep -m 1 'coreos-kernel-.*\.ebuild' | cut -d - -f 3 | cut -d . -f 1-3)
```

Also, TIL about `<ref>:<path>` for `git show`. Thanks!

Reply:
Normally it's handy for file paths to `cat` a file but it also works for directory paths